examples/pylab_examples/histogram_percent_demo.py

```diff
@@ -1,30 +1,19 @@
 import matplotlib
 from numpy.random import randn
 import matplotlib.pyplot as plt
-from matplotlib.ticker import FuncFormatter
+from matplotlib.ticker import PercentFormatter
 
 
-def to_percent(y, position):
-    # Ignore the passed in position. This has the effect of scaling the default
-    # tick locations.
-    s = str(100 * y)
-
-    # The percent symbol needs escaping in latex
-    if matplotlib.rcParams['text.usetex'] is True:
-        return s + r'$\%$'
-    else:
-        return s + '%'
-
 x = randn(5000)
 
-# Make a normed histogram. It'll be multiplied by 100 later.
-plt.hist(x, bins=50, normed=True)
+# Create a figure with some axes. This makes it easier to set the
+# formatter later, since that is only available through the OO API.
+fig, ax = plt.subplots()
 
-# Create the formatter using the function to_percent. This multiplies all the
-# default labels by 100, making them all percentages
-formatter = FuncFormatter(to_percent)
+# Make a normed histogram. It'll be multiplied by 100 later.
+plt.hist(x, bins=50, normed=True, axes=ax)
```

Comment:
I agree @madphysicist a good deal of time was spent removing the use of pylab from all the examples last year. In my opinion most of the pylab examples should be moved to other folders with more meaningful names as they are cleaned up and improved (But I am out of touch on what the current plan is for restructuring the examples)